src/views/secondary-bar/sidebar.jsx

```diff
@@ -128,7 +128,9 @@ export default function Sidebar({ expanded }) {
 			createSnackbar,
 			replaceHistory
 		);
-		const sharedItems = remove(temp, 'owner');
+		const allSharedItems = remove(temp, 'owner');
+		// Remove those share folders which broken due to revoke the rights from folder owner
+		const sharedItems = remove(allSharedItems, (item) => !item.broken);
```

Comment:
are you removing all the non-broken folders?

Reply:
Yes previously I was removing all non-broken folders and taking in new object but that changed and now just removing the broken folders from the sharedItems.